docs/proposals/network/README.md

```diff
@@ -0,0 +1,116 @@
+# Software Defined Network
+
+## Analysis
+
+### Examples
+
+- Flannel: good example, focused on networking
+- Calico
+- Weave
+- Cilium
+
+### Sources
+
+- [CNI](https://github.com/containernetworking/cni) also make a good framework for creating a new container networking project from scratch
+- [Networks - treeptik k8s](https://treeptik.gitbook.io/k8s/fundamentals/)
+- [Explain NS linux](https://www.youtube.com/watch?v=j_UUnlVC2Ss)
+- [Spec - CNI](https://www.cni.dev/docs/spec/)
+
+## Network domain
+
+### Vocabulary
+
+- *runtime* is the program responsible for executing CNI plugins
+- *plugin* is a program that applies a specified network configuration
```

Comment:
```suggestion
- *runtime* is the program responsible for executing CNI plugins.
- *plugin* is a program that applies a specified network configuration.
```

Also, what is the network configuration applied on? Is it node specific, container specific?